jmix-quartz/quartz/src/main/java/io/jmix/quartz/service/QuartzService.java

```diff
@@ -46,66 +51,82 @@ public List<JobModel> getAllJobs() {
             List<JobKey> jobDetailsKeys = jobDetailsFinder.getJobDetailBeanKeys();
 
             for (JobKey jobKey : scheduler.getJobKeys(GroupMatcher.anyJobGroup())) {
+                JobDetail jobDetail;
+                JobModel jobModel = dataManager.create(JobModel.class);
```

Comment:
Move it lower (where it begins to fill with data)

Reply:
done